chapters/stress/index.md

```diff
@@ -89,33 +118,34 @@ The parameters are organized in 4 groups-
 - Client parameters: Parameters related to how client sends data to Flow
 - Flow parameters: Host and Port of the Flow
 
-#### Sharding parameters
-
+### Sharding parameters
 
-Parallel, shards, 
+### Functional/Indexers parameter
 
-#### Functional/Indexers parameter
+## Client  
 
-Annoy, Faiss
+In order to simulate a real-world scenarios, we also test using multiple concurrent clients. This is achieved using the Python `multiprocessing` library.
 
-#### Client parameters
+In order to stress-test the system, we also issue index and query requests for several hours in a row. Each request contains a specific number of `Documents`, split across requests of the same `request_size`.
+ 
+### Advanced Indexers
 
-Concurrency of client
+As part of the benchmark, we also compare the performance of the `BaseNumpyIndexer` with the advanced indexers, based on `Annoy` and `Faiss`. We also provide specific arguments to these: `num_tress` for `Annoy`; `index_key` for `Faiss`.
```

Comment:
is not the BaseNumpyIndexer but NumpyIndexer. Annoy and Faiss also inherit from BaseNumpyIndexer